src/Microsoft.ML.StandardLearners/Standard/LinearModelParameters.cs

```diff
@@ -112,7 +111,7 @@ IEnumerator IEnumerable.GetEnumerator()
         /// <param name="weights">The weights for the linear predictor. Note that this
         /// will take ownership of the <see cref="VBuffer{T}"/>.</param>
         /// <param name="bias">The bias added to every output score.</param>
-        internal LinearPredictor(IHostEnvironment env, string name, in VBuffer<Float> weights, Float bias)
+        public LinearModelParameters(IHostEnvironment env, string name, in VBuffer<Float> weights, Float bias)
```

Comment:
>public [](start = 8, length = 6)

What is reason to change visibility?

Reply:
#1702 mentions adding public constructors for `ModelParameters`. I had this question as well - the only thing that produces `ModelParameters` is training a model, so why do we need these public. I have logged this in #1866. We can revisit this when Pete or Tom are back.